lib/src/entry.dart

```diff
@@ -47,7 +41,13 @@ class Entry {
     final fetchTime = DateTime.fromMillisecondsSinceEpoch(time, isUtc: true);
     final eTag = cached.substring(timeIndex + 1, eTagIndex);
     final configJson = cached.substring(eTagIndex + 1);
-
-    return fromConfigJson(configJson, eTag, fetchTime);
+    final Config config;
+    try {
+      final decoded = jsonDecode(configJson);
+      config = Config.fromJson(decoded);
```

Comment:
The post-processing step is necessary here as well, just like in `ConfigFetcher._deserializeConfig`.

(Unfortunately, this issue is also present in the Java and Android SDK. Please correct those as well.)